### DISCUSSION TITLE
Don't escape `appSpecs` value in `config.json`

### DISCUSSION THREAD

**Discussion by naftalibeder** [rxhanson/RectanglePro-Community#441] [Ideas]
Is it possible to add an optional unescaped layout value in the config? Something like:

```
"appSpecs" : {
  "list": [
    {
      "shortcut":  ...,
      "name": ...,
        ...
    }
  ]
}
```

I have multiple similar layouts, and it would be nice to be able to use a text editor to modify them and duplicate values between them. (Happy licensed user here! :) ).

**Comment by rxhanson** [comment]
This is unfortunately the way that the Swift json encoder/decoder works by default, and it's oddly a large enough task that a better solution would be to change the UI to make it easier to edit & duplicate. It's a task that's on my backlog.

**Reply by naftalibeder** [reply]
Understood, thanks for the reply.
